packages/block-editor/src/components/warning/style.scss

```diff
@@ -3,8 +3,7 @@
 	display: flex;
 	flex-wrap: wrap;
 	font-family: $default-font;
-	font-size: $default-font-size;
-	padding: ($grid-unit-10 - $border-width - $border-width) $grid-unit-15;
+	padding: 1em;
```

Comment:
Having padding set as an `em` unit could lead to some odd outcomes should the block be broken inside of a parent block. It will inherit its font size. May be better to use `rem` units.

Reply:
There is an open PR to switch to em units and the block Placeholder state uses em units. That's the reason for the change. Might as well be consistent.

Reply:
Thanks for the heads up....

 I wonder if that PR is considering third party blocks. This may get interesting. 🍿